openapi.yaml

```diff
@@ -737,8 +737,8 @@ components:
       description: |
         Le processus utilisé pour tricoter le tissu, parmi la liste de choix suivants :
           - mix : Tricotage moyen (mix de métiers circulaire & rectiligne)
-          - fully-fashioned : Tricotage fully-fashioned
-          - seamless : Tricotage seamless
+          - fully-fashioned : Tricotage fully-fashioned / seamless
+          - integral : Tricotage intégral / whole garment
```

Comment:
Actually it seems that the knitting process is `seamless` in English, I can't find any reference to `integral`. I think we should keep having identifiers iun English and descriptions in French, but we should definitevely NOT mix English and French in identifiers or descriptions.

Reply:
After discussion with Alban, we're leaving this as is for now, and we'll revisit the naming in the future if we get feedback from users.